sopel/modules/tell.py

```diff
@@ -8,14 +8,16 @@
 """
 from __future__ import unicode_literals, absolute_import, print_function, division
 
+import datetime
 import os
 import time
 import threading
 import sys
 
-from sopel.module import commands, nickname_commands, rule, priority, example
+from sopel.module import commands, nickname_commands, rule, priority, example, require_privilege, OP
```

Comment:
Honestly, at this point, might as well switch to `from sopel import module` and update all the usage below. This list gets absurdly long if a plugin uses even a third of the whole plugin API. 😅